Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx

```diff
@@ -354,47 +354,47 @@
     <comment>Text for Repeat</comment>
   </data>
   <data name="RepeatEvery" xml:space="preserve">
-    <value>Répétez tous les</value>
+    <value>Répéter chaque</value>
     <comment>Text for Repeat Every</comment>
   </data>
   <data name="RepeatOn" xml:space="preserve">
-    <value>Répéter sur</value>
+    <value>Répéter le</value>
     <comment>Text for Repeat On</comment>
   </data>
   <data name="Resource" xml:space="preserve">
-    <value>Ressources</value>
+    <value>Ressource</value>
     <comment>Text for Resource(s)</comment>
   </data>
   <data name="Saturday" xml:space="preserve">
-    <value>samedi</value>
+    <value>Samedi</value>
     <comment>Text for Saturday</comment>
   </data>
   <data name="Save" xml:space="preserve">
-    <value>sauver</value>
+    <value>Enregistrer</value>
     <comment>Text for Save</comment>
   </data>
   <data name="Second" xml:space="preserve">
-    <value>Seconde</value>
+    <value>Deuxième</value>
```

Comment:
seconde - this value showing in chatGPT, can check and confirm ?

Reply:
I have used all scheduler related terms and checked at once got **Deuxième**. Changed text to **Seconde**

![image](https://github.com/user-attachments/assets/3270ab14-482b-45dc-8a2a-a127a00a0ece)